packages/vaadin-overlay/src/vaadin-overlay.js

```diff
@@ -499,15 +505,7 @@ class OverlayElement extends ThemableMixin(DirMixin(PolymerElement)) {
       return;
     }
 
-    // TAB
-    if (event.key === 'Tab' && this.focusTrap && !event.defaultPrevented) {
-      // if only tab key is pressed, cycle forward, else cycle backwards.
-      this._cycleTab(event.shiftKey ? -1 : 1);
-
-      event.preventDefault();
-
-      // ESC
-    } else if (event.key === 'Escape' || event.key === 'Esc') {
+    if (event.key === 'Escape' || event.key === 'Esc') {
```

Comment:
We should check for `Escape` only as we do elsewhere. `Esc` is not a valid value for `event.key`

Reply:
Done.